time stamp
uint16[] id_list

float64[] current__mA
float64[] velocity__deg_s
float64[] position__deg
float64[] vel_trajectory__deg_s
float64[] pos_trajectory__deg
float64[] temperature__degC
float64[] input_voltage__V